dbt_semantic_interfaces/protocols/time_spine.py

```diff
@@ -28,7 +28,7 @@ def primary_column(self) -> TimeSpinePrimaryColumn:
 
     @property
     @abstractmethod
-    def custom_granularity_columns(self) -> Sequence[TimeSpineCustomGranularityColumn]:
+    def custom_granularities(self) -> Sequence[TimeSpineCustomGranularityColumn]:
```

Comment:
Don't we need to add column_name to the protocol defined on line 52?

Reply:
Oops you're right!

Reply:
Updated!